jetty-embedded-9/src/main/java/org/jboss/arquillian/container/jetty/embedded_9/ArquillianAppProvider.java

```diff
@@ -193,6 +196,8 @@ else if (context.toLowerCase(Locale.ENGLISH).startsWith("root-"))
         }
 
         webAppContext.setContextPath(context);
+        ServletHolder servletHolder = new ServletHolder(ServletMethodExecutor.ARQUILLIAN_SERVLET_NAME, ServletTestRunner.class);
```

Comment:
I would guess, as any other servlet etc etc it should be picked up here:
webAppContext.setWar(file.getAbsolutePath())

Are you forcing the Local protocol or something in your setup?